libs/application/template-api-modules/src/lib/modules/templates/driving-license-duplicate/driving-license-duplicate.service.ts

```diff
@@ -77,7 +77,7 @@ export class DrivingLicenseDuplicateService extends BaseTemplateApiService {
 
     await this.drivingLicenseService
       .drivingLicenseDuplicateSubmission({
-        pickUpLicense: pickUpLicense,
+        pickUpLicense: pickUpLicense ? true : false,
```

Comment:
This is equivalent to casting if you prefer that for readability
```suggestion
        pickUpLicense: Boolean(pickUpLicense)
```